components/UI/PortfolioItem.jsx

```diff
@@ -15,31 +15,33 @@ const PortfolioItem = (props) => {
   return (
     <div className={`${classes.portfolio__item}`}>
       <a
-        target="_blank"
-        style={{ textDecoration: "none" }}
+        target='_blank'
+        style={{ textDecoration: 'none' }}
         href={liveUrl}
```

Comment:
Please use double quotes and not single quotes as the entire codebase has double quotes. This will keep the code uniform and clean.

Reply:
Sure, I will